### ISSUE TITLE
Is the possible to speed up the inference speed?

### ISSUE THREAD

**Issue by ruanjiyang** [google-research/3d-moments#4]
Really cool project! I love it!

Just one question: Is the possible to speed up the inference speed? or any suggestion? thanks!

**Comment by qianqianwang68** [comment]
Yes, I think that would be possible.  One way would be to reduce the size of the feature extraction and image synthesis network. In addition, I haven't thoroughly profiled the inference time; it is possible that the code is not efficient and can be improved. Another way to speed up the inference without changing a lot is to remove the whole feature rendering part and directly render the color images only. See here. https://github.com/google-research/3d-moments/blob/e6903f29736a1369daf75642eacb7a8836539e77/core/renderer.py#L167 But this could also give you a performance drop.
